src/main/java/sirius/biz/web/BizController.java

```diff
@@ -267,17 +268,27 @@ protected void load(WebContext webContext, BaseEntity<?> entity, List<Mapping> p
 
     private boolean tryLoadProperty(WebContext webContext, BaseEntity<?> entity, Property property) {
         String propertyName = property.getName();
+        List<String> mlsFieldNames = property.computeAdditionalFieldNames(entity);
 
-        if (!webContext.hasParameter(propertyName) && !webContext.hasParameter(propertyName
+        if (!webContext.hasParameter(propertyName) && mlsFieldNames.isEmpty() && !webContext.hasParameter(propertyName
                                                                                + CHECKBOX_PRESENCE_MARKER)) {
             // If the parameter is not present in the request we just skip it to prevent resetting the field to null
             return true;
         }
         Value parameterValue = webContext.get(propertyName);
         try {
-            property.parseValues(entity,
-                                 Values.of(parameterValue.get(List.class,
-                                                              Collections.singletonList(parameterValue.get()))));
+            if(!mlsFieldNames.isEmpty()) {
```

Comment:
check this at the beginnig of the method so that the checkbox code etc. remains untouched